src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -203,6 +202,114 @@ describe("Editor Page Component", () => {
         expect(saveProjectSpy).toBeCalledWith(expect.objectContaining(partialProject));
     });
 
+    it("When an image is updated the asset metadata is updated", async () => {
+        const testProject = MockFactory.createTestProject("TestProject");
+        const store = createStore(testProject, true);
+        const props = MockFactory.editorPageProps(testProject.id);
+        const wrapper = createComponent(store, props);
+        const imageAsset = testAssets[0];
+
+        await MockFactory.flushUi();
+        wrapper.update();
+
+        const editedImageAsset: IAssetMetadata = {
+            asset: imageAsset,
+            regions: [MockFactory.createMockRegion()],
+        };
+
+        const saveMock = assetServiceMock.prototype.save as jest.Mock;
+        saveMock.mockClear();
```

Comment:
why do we have to call `mockClear` when the mock is brand new?

Reply:
The mock isn't brand new, its just being casted to a proper mock type after being instantiated up above in the before Each.

Reply:
And we want to reset the mock call count since by default the editor page selects the first asset in the list.